api/types/stats.go

```diff
@@ -89,10 +89,15 @@ type NetworkStats struct {
 
 // Stats is Ultimate struct aggregating all types of stats of one container
 type Stats struct {
-	Read        time.Time    `json:"read"`
-	Network     NetworkStats `json:"network,omitempty"`
-	PreCPUStats CPUStats     `json:"precpu_stats,omitempty"`
-	CPUStats    CPUStats     `json:"cpu_stats,omitempty"`
-	MemoryStats MemoryStats  `json:"memory_stats,omitempty"`
-	BlkioStats  BlkioStats   `json:"blkio_stats,omitempty"`
+	Read time.Time `json:"read"`
+
+	// Network is for fallback stats where API Version <= 1.21
+	Network NetworkStats `json:"network,omitempty"`
```

Comment:
I do considered to use the similar solution as what the `ContainerJSON` does. But later I found it is  complicated, and provides not obvious value, it will introduce many things like:
```
 if version.LessThan("1.xx")  {
 ...
 }
```
which I don't thinks is a good code style, and it will force the relative functions to add arguments `version` and maybe more things.

Besides the word `Network` explain itself well and so does the `Networks`.  When someday we decide to drop `Network`, it is fine to just remove it and it will not need to remove those
```
if version <=
```

If you still prefer the old ways, it's OK for me.